distributed/shuffle/_worker_plugin.py

```diff
@@ -855,7 +856,7 @@ async def _(
         self._runs.add(shuffle)
         return shuffle
 
-    async def close(self) -> None:
+    async def teardown(self, worker: Worker) -> None:
```

Comment:
Along with this so that the worker plugin now conforms to the `WorkerPlugin` interface.